LaraApi/src-lara/lara/cmake/CMaker.js

```diff
@@ -1,422 +1,423 @@
 laraImport("lara.cmake.CMakerUtils");
```

Comment:
Are most of these changes because of formatting? What actual changes occurred, it was only to make the class extend Tool?

The JavaScript formatter to be used is Prettier with default options.

Reply:
It is odd that it changed so much, and it could have been my local formatter. I think it would be interesting if we had a GitHub action that formatted the code with Prettier for every pull request: https://peterevans.dev/posts/github-actions-how-to-automate-code-formatting-in-pull-requests/